gateway-js/src/__tests__/gateway/lifecycle-hooks.test.ts

```diff
@@ -67,38 +71,6 @@ describe('lifecycle hooks', () => {
     await gateway.stop();
   });
 
-  it('calls experimental_didFailComposition with a bad config', async () => {
```

Comment:
This is the only "breaking change" I'm aware of, and it's an experimental config option. I could wire this option back up to the relevant `SupergraphManager`s for now if this is a bad change.

Reply:
Is the change that this hook doesn't work at all any more? If so probably remove it from GatewayConfigBase so that people using it get compilation failures instead of code that silently does less than it used to? We can add it back if we get complaints.